web/themes/custom/kitchen_karaoke/components/01-atoms/buttons/_buttons.scss

```diff
@@ -98,3 +98,40 @@ $button-colors: (
     @include button-large;
   }
 }
+
+.btn-primary {
```

Comment:
A note that this class isn't really used anywhere now and I couldn't find it on the old one so it may be that there was something with the bootstrap base that we are losing there. I also doubt that we'll want the exact button style.